data_handler/handlers/liquidable_debt/debt_handlers.py

```diff
@@ -275,3 +275,109 @@ def calculate_liquidable_debt(self, protocol_name: str = None) -> list:
                 )
 
         return result_data
+
+
+class HashstackV0DBLiquidableDebtDataHandler(BaseDBLiquidableDebtDataHandler):
+    """
+    A Hashstack_V0 handler that collects data from the DB,
+    computes the liquidable debt and stores it in the database.
+
+    :cvar AVAILABLE_PROTOCOLS: A list of all available protocols.
+    """
+
+    def __init__(
+            self,
+            loan_state_class: Type[State],
+            loan_entity_class: Type[LoanEntity],
+    ):
+        super().__init__()
+        self.state_class = loan_state_class
+        self.loan_entity_class = loan_entity_class
+
+    def fetch_data(self, protocol_name: ProtocolIDs | str) -> tuple:
+        """
+        Prepares the data for the given protocol.
+        :param protocol_name: Protocol name.
+        :return: tuple
+        """
+        loan_data = self.db_connector.get_loans(
+            model=HashtackCollateralDebt,
```

Comment:
instead of `HashtackCollateralDebt` should be `LoanState`

Reply:
But `LoanState` doesn't have `debt_category` field which I need to use below.

Reply:
In this case, you need to split this loginc into two steps.
Becausue `LoanState` and `HashtackCollateralDebt` have different purpose of usage

Reply:
> In this case, you need to split this loginc into two steps

Is your approach performant? I mean in this case I need to fetch all loan states from `LoanState` and than for each `LoanState` instance to get `debt_category` from `HashtackCollateralDebt`. Is it really good idea?

Reply:
Done